### DISCUSSION TITLE
CSS Conflicts

### DISCUSSION THREAD

**Discussion by abbasmashaddy72** [Power-Components/livewire-powergrid#680] [Q&A]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### PowerGrid

3.5

### Laravel

9.19

### Livewire

2.10

### Alpine JS

3.10.4

### Theme

Tailwind 3.x with tailwind/forms

### Describe the bug.

The Issue is with the dropdowns CSS

I had imported the CSS manually from the package and made the changes to work

now it is working

### To Reproduce...

Followed the Docs imported in my theme

**Comment by luanfreitasdev** [comment]
Can you show how it was before, and what you did to fix the problem? If you can show a screenshot it would be better.
